src/components/errorboundary/errorboundary.jsx

```diff
@@ -46,7 +50,8 @@ class ErrorBoundary extends React.Component {
     }
 }
 ErrorBoundary.propTypes = {
-    children: PropTypes.node
+    children: PropTypes.node,
+    name: PropTypes.string
```

Comment:
This is picky, but maybe this should be called componentName (or something like that) to indicate that we expect it to be the name of the component that the error originates from since the sentry tag is "component"?

Reply:
Good idea, doing it

Reply:
Done